.github/workflows/benchmark_visualization.yml

```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   benchmark:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ 'github.event.pull_request.head.repo.fork '  && 'ubuntu-22.04' || 'codebuild-soci-visualization-workflow-${{ github.run_id }}-${{ github.run_attempt }}-ubuntu-7.0-xlarge' }}
```

Comment:
Based on the tests run in this PR it seems this will just always run 22.04.

Also, I assume we're switching to 22.04 because that's what CodeBuild uses?

Reply:
Updated the workflow, there is not really a simpler way to add conditions to the runner type. One of those times where WET code makes more sense to reduce complicated conditions.